src/loki/search.cc

```diff
@@ -418,19 +420,22 @@ struct bin_handler_t {
       PathLocation::PathEdge path_edge{candidate.edge_id, length_ratio, candidate.point,
                                        distance,          side,         reach.outbound,
                                        reach.inbound,     angle};
-      // correlate the edge we found
-      if (side_filter(path_edge, location, reader) || heading_filter(location, angle) ||
-          layer_filter(location, layer)) {
+      // correlate the edge we found if its not filtered out
```

Comment:
ok so here inside `correlate_edge` we still need to re-evaluate the search filter because its possible that one of the edges is only filtered because its a traffic closure, thus leaving only one of the directions of the edge pair open to this location. it shouldnt be possible to get here in a case where both directions are filtered or otherwise not allowed so i didnt had a check for it. if it happens, and it shouldnt (would be a logic error), the result will be no candidates for a given location (and hopefully a bug report)